lib/usecases/params/forgot_password_params.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:survey_flutter_ic/usecases/params/forgot_password_user_params.dart';
+
+class ForgotPasswordParams {
+  final ForgotPasswordUserParams user;
```

Comment:
I think we might not need to make it complex like this, the param should contain `email` field as String only, and then in the request model, we will wrap it later on with `user` field. As you can see, the repository method `forgotPassword` only requires `email` field only so wrap it under `user` field like this does not make sense to me 😄 
```suggestion
  final String email;
```

Reply:
Or we can even use `String email` directly with the params in `ForgotPasswordUseCase`, so that we don't need to create a separate params class like this because the `ForgotPasswordUseCase` only require `email` field which is not complex as `LoginUseCase` as it required 2 params `email` and `password`
```
class ForgotPasswordUseCase extends UseCase<ForgotPasswordModel, String>
```

Reply:
I think we don't need this class `ForgotPasswordParams` anymore.